exampleCourse/questions/element/codeDocumentation/question.html

```diff
@@ -13,12 +13,7 @@ <h2>Submission Elements</h2>
 
 <hr />
 
-<div class="card my-2">
-    <div class="card-header">
-        <code>pl-multiple-choice</code>
-    </div>
-
-    <div class="card-body">
+<pl-card header="<code>pl-multiple-choice</code>">
```

Comment:
It's usually a good idea to escape HTML inside arguments.
```suggestion
<pl-card header="&lt;code&gt;pl-multiple-choice&lt;/code&gt;">
```
Same with other similar parts.

Reply:
I think the intent here is to have `pl-multiple-choice` appear as code (i.e. using the HTML here for styling), so escaping would break this.

Reply:
@eliotwrobson is correct that the intent is to appear as code but it will still render if it is escaped.

<details>
<summary>Example</summary>

![image](https://github.com/user-attachments/assets/cdf08617-7dbf-43bb-acc1-b81a74ca6c45)

```
<pl-card header="&lt;code&gt;pl-multiple-choice&lt;/code&gt;">

    <pl-question-panel>
      Example of a multiple choice element.
      <p> Correct Input: <code>negative</code> </p>
    </pl-question-panel>
    <pl-multiple-choice answers-name="acc">
      <pl-answer correct="false">positive</pl-answer>
      <pl-answer correct="true">negative</pl-answer>
      <pl-answer correct="false">zero</pl-answer>
    </pl-multiple-choice>
</pl-card>

<pl-card header="<code>pl-checkbox</code>">
```

</details>

I think it is neater showing it this way for clarity of how it works but can update them to the alternative if there is a strong preference

Reply:
The escaping is not intended to cause the code to show as escaped, but to ensure the argument is parsed without errors. The same thing happens in popover cases where the content of the popover is HTML, you also need to escape it in the argument.

Reply:
Ahh I see, if that is HTML best practice then I'd prefer to have the escaping.

Reply:
Think of cases where the HTML inside the argument itself has an argument with quotes, that could cause some issues.

Reply:
Updated in https://github.com/PrairieLearn/PrairieLearn/pull/11382/commits/c93004d8103fa331cd27294bf8454f669e04cb9b